x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/components/configuration/adaptive_allocations_title.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useState } from 'react';
+import {
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+export const AdaptiveAllocationsTitle: FC = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  return (
+    <>
+      <EuiFlexGroup alignItems="center" gutterSize="none">
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="xxs" data-test-subj="max_number_of_allocations-details-label">
```

Comment:
We usually follow camelCase notation for test subjects

Reply:
Updated in ebff5bc61d7b5212ac76edd658fff6c6f2a44c42